dtool/src/interrogate/interfaceMaker.cxx

```diff
@@ -665,7 +665,6 @@ record_object(TypeIndex type_index) {
 
   InterrogateDatabase *idb = InterrogateDatabase::get_ptr();
   const InterrogateType &itype = idb->get_type(type_index);
-  assert(&itype != NULL);
```

Comment:
It seems like we should be asserting that the database has the type before trying to get it, or having get_type return a pointer if it could be "nullable." Running around with a reference to a dereferenced NULL is a Bad Thing(TM).

Reply:
Yes, I agree with this.

Reply:
Resolution: remove